### PR TITLE
added responsiveness

### PR DESCRIPTION
**Pull request by Raghav888**

### CODE REVIEW

File: documentation/document.css [hunk 1]
```diff
@@ -179,4 +179,28 @@
...
         color: black
+    }
+    
+    @media only screen and (max-width:930px) {
+        html {
+            font-size: 50%;
+        }
+    }
+    
+    @media only screen and (max-width: 768px) {
```
Comment (RohitCoder23) [inline documentation/document.css:190]: Add the responsiveness in index page too, rest look good

**Review by RohitCoder23 — COMMENTED** [review]
Good to merge

**Comment by RaviShrivastva** [comment]
Nice work. Please new live demo link in next PR

File: documentation/document.css (hunk 1 above)
Comment (Raghav888) [reply documentation/document.css:190]: Yes, will do and raise new PR

**Comment by Raghav888** [comment]
> Nice work. Please new live demo link in next PR

Yes will add

**Pull request merged**